### ISSUE TITLE
Adding in linebreaks filters

### ISSUE THREAD

**Issue by Zerthick** [chakki-works/doccano#27]
I'm attempting to add in linebreak filters to allow newlines to be rendered within documents to complement #17 however I can't seem to get them working.

I've identified the line which inserts the document text to be line 34 of document_classification.html
https://github.com/chakki-works/doccano/blob/master/app/server/templates/annotation/document_classification.html#L34

However modifying it to `[[ docs[pageNumber].text|linebreaks ]]` does not produce the desired effect.

On a side note I am not very familiar with Django, what is the significance of the double square-bracket syntax? I have not seen it in any Django examples.  Aren't variables typically inserted with double curly-brackets in templates?

**Comment by Zerthick** [comment]
I've managed to address the issue by modifying the line to:
```
<span style="white-space: pre-wrap;" id="myspan">[[ docs[pageNumber].text ]]</span>
```

I've noticed you and an issue with vue regarding the spacing of in-line content. Vue is preserving the spacing of the content within the template so changing the above line to something like

```
<span style="white-space: pre-wrap;" id="myspan">
    [[ docs[pageNumber].text ]]
</span>
```

Causes the text to become indented when rendered in the html page.

**Comment by BrambleXu** [comment]
Hi @Zerthick, you mentioned document_classification.html, which means you create a document classification project? Could you give me more detail about what project you use, what file format you upload, and the problem you meet. It would be helpful if you could share your demo json file.

**Comment by BrambleXu** [comment]
Thanks for your code, @Zerthick. I test it and it works well for document classification task and seq2seq task while annotating. But the exporting csv file will render the newline and this will cause one sentence seperate to two lines. In order to avoid this situation, I will provide the option to export json file. #15

**Comment by Zerthick** [comment]
@BrambleXu Added some more comments to https://github.com/chakki-works/doccano/pull/34

**Comment by BrambleXu** [comment]
Thanks, @Zerthick. The information you provide is very useful!

**Comment by BrambleXu** [comment]
Solved in #37

**Comment by BrambleXu** [comment]
Hi, @Zerthick. In the upcoming JSON export feature, even if we won't rendering line breaks in annotation page for sequence labeling task, the exported JSON file will contain linebreaks.
